fedmsg_meta_fedora_infrastructure/buildsys.py

```diff
@@ -274,5 +281,7 @@ def objects(self, msg, **config):
                 'tags',
                 msg['msg'].get('tag', 'unknown'),
             ])])
-        else:
-            raise NotImplementedError("%r" % msg)
+        elif 'buildsys.rpm.sign' in msg['topic']:
+            return set(['signatures/' + msg['msg']['info']['name']])
+
```

Comment:
But.. there's a ``return set()`` at the bottom?

Either way, if it returns ``None`` (like it would if there's no return statement) there is logic in the code that calls this function that knows that ``set()`` should be the default value if no explicit value is returned.